internal/driver/gomobile/driver.go

```diff
@@ -63,7 +63,15 @@ func (d *mobileDriver) currentWindow() fyne.Window {
 		return nil
 	}
 
-	return d.windows[len(d.windows)-1]
+	var last fyne.Window
+	for i := len(d.windows) - 1; i >= 0; i-- {
+		last = d.windows[i]
+		if last.(*window).visible {
+			return last
+		}
+	}
+
+	return last
```

Comment:
This code will return the last hidden window if they are all hidden - is that intended?
Above we return nil if there are no windows, I think we should do that here too

Reply:
> This code will return the last hidden window if they are all hidden - is that intended?

Yes, that was intended.

> Above we return nil if there are no windows, I think we should do that here too

That was something that I was not sure. In mobile, if all the windows are hidden, what does it mean? 🤔

Reply:
It should not be done - app will be unusable. But not sure that the most recently added window is the right response in this situation... It should never happen, but there is specifically code in their to accommodate it so I figured we should be sure that is what should occur if it can.

Reply:
> But not sure that the most recently added window is the right response in this situation...

Indeed this change is returning the parent window (not the most recently added) because I am looping in reverse. Now I realized I have misread your first comment. It is not the last hidden window but the first one (parent one).

Reply:
Oh yes I see, sorry for misreading the first time